### PR TITLE
added `Wikipedia` document loader

### PR DESCRIPTION
**Pull request by leo-gan**
- Added the `Wikipedia` document loader. It is based on the existing `unilities/WikipediaAPIWrapper`
- Added a respective ut-s and example notebook
- Sorted list of classes in __init__

### CODE REVIEW

**Review by hwchase17 — APPROVED** [review]
this is great - thanks!

File: langchain/document_loaders/__init__.py [hunk 1]
```diff
@@ -120,13 +119,15 @@
     "BlockchainDocumentLoader",
     "CSVLoader",
     "ChatGPTLoader",
+    "ChatGPTLoader",
```
Comment (dev2049) [inline langchain/document_loaders/__init__.py:122]: duplicate?

Comment (leo-gan) [reply langchain/document_loaders/__init__.py]: [x]

File: langchain/utilities/wikipedia.py [hunk 2]
```diff
@@ -41,23 +47,70 @@ def validate_environment(cls, values: Dict) -> Dict:
...
-            wiki_page = self.wiki_client.page(title=page, auto_suggest=False)
-            return f"Page: {page}\nSummary: {wiki_page.summary}"
+            return self.wiki_client.page(title=page, auto_suggest=False)
         except (
             self.wiki_client.exceptions.PageError,
             self.wiki_client.exceptions.DisambiguationError,
         ):
             return None
+
+    def load(self, query: str) -> List[Document]:
```
Comment (dev2049) [inline langchain/utilities/wikipedia.py:102]: tangent: @hwchase17 would make for a cool Retriever as well, maybe worth coming up with a little demo (in a sep pr)

**Review by hwchase17 — APPROVED** [review]

**Pull request merged**